modules/virt-about-vm-snapshots.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-about-vm-snapshots_{context}"]
+= About virtual machine snapshots
+
+A _snapshot_ represents the state and data of a virtual machine (VM) at a specific point in time. You can use a snapshot to restore an existing VM to a previous state (represented by
+the snapshot) for backup and disaster recovery or to rapidly roll back to a previous development version.
+
+An offline VM snapshot is created from a VM that is powered off (Stopped state). The snapshot stores a copy of each Container Storage Interface (CSI) volume attached to the VM and a copy of the VM
+specification and metadata. Snapshots cannot be changed after creation.
+
+With the offline VM snapshots feature, a cluster administrator and an application developer can:
```

Comment:
Might read better as plural: "... cluster administrators and application developers can..."